### PR TITLE
chore: added more hba1c NPU codes to the fake dataset

### PR DESCRIPTION
**Pull request by lwjohnst86**
These codes are for HbA1c, so want to inflate the number of values within the fake data

### CODE REVIEW

**Review by signekb — APPROVED** [review]
Nice 🎉

File: data-raw/simulate-data.R [hunk 1]
```diff
@@ -185,12 +185,20 @@ insert_false_drug_names <- function(data, proportion = 0.05) {
...
+insert_analysiscode <- function(data, proportion = 0.3) {
+  # NPU27300: New units for HbA1c
+  # NPU03835: Old units for HbA1c
+  data |>
+    dplyr::mutate(
+      dplyr::across(
+        dplyr::matches("^analysiscode$"),
+        \(column) dplyr::if_else(
+          runif(dplyr::n()) < proportion,
+          sample(c("NPU27300", "NPU03835"), 1),
```
Comment (Aastedet) [inline data-raw/simulate-data.R:197]:
```suggestion
          sample(c("NPU27300", "NPU03835"), dplyr::n(), replace = TRUE),
```

**Review by Aastedet — CHANGES_REQUESTED** [review]
Small edit to make mutated rows a mix of 27300 and 03835, instead of only-27300 or only-03835 depending on the random seed.

File: data-raw/simulate-data.R (hunk 1 above)
Comment (lwjohnst86) [reply data-raw/simulate-data.R]: Nice catch!

**Pull request merged**